GVRf/Framework/framework/src/main/java/org/gearvrf/GVRCamera.java

```diff
@@ -309,6 +309,15 @@ public int getChildrenCount() {
         return 0;
     }
 
+    public void setViewMatrix(float[] matrix)
```

Comment:
I added a JavaDoc for this. It is public and is used to set up the view matrix for a camera that is not attached to a scene object so it can render to a  texture.